src/main/thrift/parquet.thrift

```diff
@@ -582,6 +582,12 @@ union BloomFilterHash {
   /** xxHash Strategy. **/
   1: XxHash XXHASH;
 }
+
+/**
+ * The compression used in the Bloom filter.
+ **/
+union BloomFilterCompression {}
```

Comment:
I think there should be a `BloomFilterCompression` for `Uncompressed` so that implementations can check for it and ignore bloom filters if the compression doesn't match.

Reply:
Agreed; done.